docs/pages/repo/docs/core-concepts/caching/to-cache-or-not-to-cache.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: To Cache or Not to Cache
+description: Whether 'tis more performant to cache.
+---
+
+import Callout from "../../../../../components/Callout";
+
+# To Cache or Not To Cache
```

Comment:
What do you think about starting off with a sentence recommending caching by default? A user who wants a quick answer to "should I cache or not" should probably get a "yes, cache" and then revisit for more details if something goes wrong.